meshnets/utils/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""Init"""
+from .callbacks import (MLFlowLoggerCheckpointer,
+                        MLFlowLoggerFinalizeCheckpointer, GPUUsage,
+                        GPUUsageMean, GradientNorm, GeometricBatchSize)
+from .data_loading import (load_edge_mesh_pv, edges_from_meshio_mesh,
+                           load_edge_mesh_meshio, load_triangle_mesh)
+from .data_processing import (edge_mesh_to_graph, triangle_mesh_to_graph,
+                              mesh_file_to_graph_data)
+from .data_visualization import (plot_mesh, plot_mesh_with_scalars,
+                                 plot_mesh_comparison)
+from .datasets import FromDiskGeometricDataset
+from .model_loading import load_model_from_mlflow
+from .model_training import train_model
```

Comment:
I think in this particular file you want to do e.g. `from . import callbacks`, as not to expose all utilities equally, e.g. `meshnets.utils.FromDiskGeometricDataset` and `meshnets.utils.GradientNorm`. I think it's useful to keep the submodule names, e.g. `meshnets.utils.datasets.FromDiskGeometricDataset` for context.

Reply:
Done
![image](https://github.com/inductiva/meshnets/assets/33664950/155efc44-0a51-4616-9463-7439606e54b9)